ReactiveCocoa/RACObjCRuntimeUtilities.h

```diff
@@ -1,6 +1,9 @@
 #import <Foundation/Foundation.h>
+#import <objc/runtime.h>
 
 NS_ASSUME_NONNULL_BEGIN
+void _rac_objc_setAssociatedObject(const void* object, const void* key, id _Nullable value, objc_AssociationPolicy policy);
```

Comment:
This could use documentation about why it's necessary (if we continue to use it).